karma.conf.js

```diff
@@ -44,6 +45,14 @@ module.exports = function(config) {
         },
         webpack: {
             mode: "none",
+            plugins: [
+                new NodePolyfillPlugin()
+            ],
+            resolve: {
+                alias: {
+                    "../../../lib/linter$": "../../../build/eslint.js"
```

Comment:
Since we are now bundling `build/eslint.js` into tests, do we need to load it in the browser (i.e. can we remove it from the `files` array above)?

Reply:
Are there any pros?

Reply:
Technically not much if any, but for a better understanding of what's happening here, so that someone who reads the code doesn't have to think about why are we loading the script and where is it used (while it actually isn't used, the code is bundled in another script). Is there a benefit of [loading build/eslint.js](https://github.com/eslint/eslint/blob/61894247977888e66e6b9146499a9ae111e924bb/karma.conf.js#L29) now?